docs/content/faq.md

```diff
@@ -87,3 +87,27 @@ Which currently includes:
 - hg
 - bzr
 - fossil
+
+### When should I use a vendor directory, and when should I use Athens?
+
+The Go community has used vendor directories for a long time before module proxies like Athens came along, so naturally each group collaborating on code should decide for themselves whether they want to use a vendor directory, use Athens, or do both!
+
+Vendor (without a proxy) are valuable when:
```

Comment:
Can you write this as `Using a vendor directory (without a proxy) is valuable when:`?